tests/e2e/utils/lib.js

```diff
@@ -1,12 +1,5 @@
-export const waitForSelector = async function ( page, selector ) {
-	return await page.evaluate(
-		( selector ) => Boolean( document.querySelector( selector ) ),
-		selector
-	);
-};
-
 export const waitForElementCount = async function ( page, domSelector, count ) {
```

Comment:
Looks like we can remove the `async` here.

Reply:
Ha good spotting. We should probably run lint over this code